CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 * Core: Enabled Cluster Mode periodic checks by default ([#1089](https://github.com/aws/glide-for-redis/pull/1089))
 * Node: Added Rename command. ([#1124](https://github.com/aws/glide-for-redis/pull/1124))
 * Python: Added JSON.TOGGLE command ([#1184](https://github.com/aws/glide-for-redis/pull/1184))
+* Java: Added ZMSCORE command (TODO ADD HERE PR #)
```

Comment:
No need to mention java here, but maybe core. awaiting for AWS team to reply whether we need to record that here or not.